scripts/install/frontend.sh

```diff
@@ -10,6 +10,7 @@ function install_deps() {
     echo
     npm install;
     npm install brunch canvas vega @vue/cli;
+    npm run build; 
```

Comment:
Please undo this change. Unless we are deploying for production, there is no reason we need to build the frontend statically; given that installing for dev vs. prod does not exist right now, it doesn't make sense to do this yet.

Reply:
So, I do `install-dev` on most of our instances because we use them for debugging.  I can do `npm run build` myself in those cases. I see your point.